Modules/_json.c

```diff
@@ -1251,6 +1251,25 @@ encoder_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
     return (PyObject *)s;
 }
 
+static PyObject *
+_create_newline_indent(PyObject* indent, Py_ssize_t indent_level)
+{
+    PyObject* current_indent = PySequence_Repeat(indent, indent_level);
+    if (current_indent == NULL) {
+        return NULL;
+    }
+    PyObject* start = PyUnicode_FromOrdinal(10);
```

Comment:
Not everyone might know that this is a newline:

```suggestion
    PyObject* start = PyUnicode_FromOrdinal(10);  // newline
```

Reply:
Does `'\n'` work here, maybe with a cast? chars in C are a kind of number.

Reply:
Yes.

Reply:
You can resolve this.